sound/soc/intel/common/soc-acpi-intel-cnl-match.c

```diff
@@ -76,7 +76,7 @@ struct snd_soc_acpi_mach snd_soc_acpi_intel_cnl_sdw_machines[] = {
 		.link_mask = 0xB, /* 3 active links required */
 		.drv_name = "sdw_rt711_rt1308_rt715",
 		.sof_fw_filename = "sof-cnl.ri",
-		.sof_tplg_filename = "sof-cml-rt711-rt1308-rt715.tplg",
+		.sof_tplg_filename = "sof-cml-rt711-rt1308-mono-rt715.tplg",
```

Comment:
ok, but this will not work for the DVT1 since all 4 links are enabled. Not sure about the way to proceed if the link information is bad.

Reply:
@plbossart maybe we need some tplg name fixup based on what codecs are attached like what we do with DMIC's for HDA platforms? Just thinking out loud. Not really sure about what it really means in SDW jargons.

Reply:
> ok, but this will not work for the DVT1 since all 4 links are enabled. Not sure about the way to proceed if the link information is bad.

@plbossart Actually, QA and I are testing with options soundire_intel_init sdw_link_mask=0xb

Reply:
ok, so we really need to improve the detection and scan the links to make sure they contain expected codecs.